bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/messages.properties

```diff
@@ -493,12 +493,13 @@ OpenPerspectiveDialogAction_tooltip=Open Perspective
 
 #---- General Preferences----
 PreferencePage_noDescription = (No description available)
-PreferencePageParameterValues_pageLabelSeparator = \ >\ 
+PreferencePageParameterValues_pageLabelSeparator = \ >\
 ThemingEnabled = E&nable theming
 ThemeChangeWarningText = Restart for the theme changes to take full effect
-ThemeChangeWarningTitle = Theme Changed 
+ThemeChangeWarningTitle = Theme Changed
 # --- Workbench -----
 WorkbenchPreference_openMode=Open mode
+WorkbenchPreference_closeMode=Close mode
```

Comment:
we don't need this text any more. Pls. remove it here and from the messages class

Reply:
id did remove it